qa-mickey.planx-pla.net/manifest.json

```diff
@@ -30,6 +28,7 @@
     "sower": "quay.io/cdis/sower:2020.08",
     "spark": "quay.io/cdis/gen3-spark:2020.08",
     "tube": "quay.io/cdis/tube:2020.08",
+    "ws-storage": "quay.io/cdis/ws-storage:master",
```

Comment:
there is `2021.01` release for wss if you want to use

Reply:
I will update to 2021.01 later for all the services. 👍